server/implementation/src/main/java/io/smallrye/graphql/execution/error/ExecutionErrorsService.java

```diff
@@ -86,12 +86,25 @@ private JsonObject toJsonError(GraphQLError error) {
 
         addKeyValue(objectBuilder, EXCEPTION, exception.getClass().getName());
         addKeyValue(objectBuilder, CLASSIFICATION, error.getErrorType().toString());
+        addKeyValue(objectBuilder, CODE, toErrorCode(exception));
         Map<String, Object> extensions = error.getExtensions();
         populateCustomExtensions(objectBuilder, extensions);
 
         return Optional.of(objectBuilder.build());
     }
 
+    private String toErrorCode(Throwable exception) {
```

Comment:
Please add a TODO that this should change to use the Model and the model needs to get this using Jandex.

Reply:
done